model/__init__.py

```diff
@@ -335,73 +340,66 @@ def initialize(cls, url, create_materialized_work_class=True):
             sql = open(resource_file).read()
             connection.execute(sql)
 
+        session = Session(connection)
+        cls.initialize_data(session)
+
         if connection:
             connection.close()
 
-        if create_materialized_work_class:
-            from licensing import LicensePool
-            class MaterializedWorkWithGenre(Base, BaseMaterializedWork):
-                __table__ = Table(
-                    cls.MATERIALIZED_VIEW_LANES,
-                    Base.metadata,
-                    Column('works_id', Integer, primary_key=True, index=True),
-                    Column('workgenres_id', Integer, primary_key=True, index=True),
-                    Column('list_id', Integer, ForeignKey('customlists.id'),
-                           primary_key=True, index=True),
-                    Column(
-                        'list_edition_id', Integer, ForeignKey('editions.id'),
-                        primary_key=True, index=True
-                    ),
-                    Column(
-                        'license_pool_id', Integer,
-                        ForeignKey('licensepools.id'), primary_key=True,
-                        index=True
-                    ),
-                    autoload=True,
-                    autoload_with=engine
-                )
-                license_pool = relationship(
-                    LicensePool,
-                    primaryjoin="LicensePool.id==MaterializedWorkWithGenre.license_pool_id",
-                    foreign_keys=LicensePool.id, lazy='joined', uselist=False)
-
-            globals()['MaterializedWorkWithGenre'] = MaterializedWorkWithGenre
-            import core.model
-            core.model.MaterializedWorkWithGenre = MaterializedWorkWithGenre
-
```

Comment:
This chunk is no longer necessary because MaterializedWorkWithGenre is defined at compile time, in model/works.py